tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot

```diff
@@ -204,6 +205,41 @@ Verify Alert "RHODS Probe Success Burn Rate" Is Fired When RHODS Dashboard Is Do
 
     [Teardown]    ODS.Restore Default Deployment Sizes
 
+JupyterHub Image Builds Are Failing
+    [Tags]    Sanity    
+    ...     Tier2    
+    ...     ODS-717
+    ${build_name_fail} =    Bring JupyterHub Image Builds Are In Failing State
+    Prometheus.Wait Until Alert Is Firing    ${RHODS_PROMETHEUS_URL}
+    ...    ${RHODS_PROMETHEUS_TOKEN}
+    ...    Builds
+    ...    JupyterHub image builds are failing
+    ${build_name} =    Start New Build    namespace=redhat-ods-applications
+    ...    buildconfig=s2i-tensorflow-gpu-cuda-11.4.2-notebook
+    Prometheus.Wait Until Alert Is Not Firing    ${RHODS_PROMETHEUS_URL}
+    ...    ${RHODS_PROMETHEUS_TOKEN}
+    ...    Builds
+    ...    JupyterHub image builds are failing
+    [Teardown]    Delete Build Using Name    namespace=redhat-ods-applications    build_name=${build_name_fail}
+
+JupyterHub Image Builds Are Failing And Fires Alert For 30 min
+    [Tags]    Sanity    
+    ...     Tier2    
+    ...     ODS-790
+    ${build_name_fail} =    Bring JupyterHub Image Builds Are In Failing State
+    Prometheus.Wait Until Alert Is Firing    ${RHODS_PROMETHEUS_URL}
+    ...    ${RHODS_PROMETHEUS_TOKEN}
+    ...    Builds
+    ...    JupyterHub image builds are failing
+
+    Prometheus.Wait Until Alert Is Not Firing    ${RHODS_PROMETHEUS_URL}
+    ...    ${RHODS_PROMETHEUS_TOKEN}
+    ...    Builds
+    ...    JupyterHub image builds are failing
+    ...    32 min
+    ${build_name} =    Start New Build    namespace=redhat-ods-applications
+    ...    buildconfig=s2i-tensorflow-gpu-cuda-11.4.2-notebook
+    [Teardown]    Delete Build Using Name    namespace=redhat-ods-applications    build_name=${build_name_fail}
```

Comment:
Wouldn't this leave the cluster without a tensorflow build?

Reply:
we are deleting failed builds and on the cluster, we are placing good one build